server/src/test/java/org/elasticsearch/cluster/ClusterHealthIT.java

```diff
@@ -250,15 +257,35 @@ public void clusterStateProcessed(String source, ClusterState oldState, ClusterS
     }
 
     @Test
+    @UseRandomizedSchema(random = false)
     public void testHealthOnMasterFailover() throws Exception {
         final String node = internalCluster().startDataOnlyNode();
+        boolean withIndex = randomBoolean();
```

Comment:
tested locally with both hard-coded true/false values, used to fail because of missing https://github.com/elastic/elasticsearch/commit/32fa90c9bac0355680c19f2f76d78aa9af2a4ba0